build/codegen/codegen.sh

```diff
@@ -24,9 +24,12 @@ cd ${scriptdir}/../../vendor/k8s.io/code-generator && ./generate-groups.sh \
 # this seems busted in the release-1.8 branch
 #  --go-header-file ${SCRIPT_ROOT}/build/codegen/header.txt
 
+# Based on https://stackoverflow.com/a/38595160 to fix GNU and BSD issues
```

Comment:
Rather than the link, could you just provide a short explanation? Just a few sentences.